diff --git a/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/MiscRuntimeServices/BlackBoxTest/MiscRuntimeServicesBBTestConformance.c b/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/MiscRuntimeServices/BlackBoxTest/MiscRuntimeServicesBBTestConformance.c
index e685fac0..8a3d3333 100644
--- a/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/MiscRuntimeServices/BlackBoxTest/MiscRuntimeServicesBBTestConformance.c
+++ b/uefi-sct/SctPkg/TestCase/UEFI/EFI/RuntimeServices/MiscRuntimeServices/BlackBoxTest/MiscRuntimeServicesBBTestConformance.c
@@ -97,6 +97,8 @@ BBTestUpdateCapsuleConformanceTest (
   	             );
   if (Status == EFI_INVALID_PARAMETER) {
   	AssertionType = EFI_TEST_ASSERTION_PASSED;
+  } else if (Status == EFI_UNSUPPORTED) {
+        AssertionType = EFI_TEST_ASSERTION_WARNING;
   } else {
     AssertionType = EFI_TEST_ASSERTION_FAILED;
   }
@@ -130,6 +132,8 @@ BBTestUpdateCapsuleConformanceTest (
   	             );
   if (Status == EFI_INVALID_PARAMETER) {
   	AssertionType = EFI_TEST_ASSERTION_PASSED;
+  } else if (Status == EFI_UNSUPPORTED) {
+        AssertionType = EFI_TEST_ASSERTION_WARNING;
   } else {
     AssertionType = EFI_TEST_ASSERTION_FAILED;
   }
@@ -164,6 +168,8 @@ BBTestUpdateCapsuleConformanceTest (
   	             );
   if (Status == EFI_INVALID_PARAMETER) {
   	AssertionType = EFI_TEST_ASSERTION_PASSED;
+  } else if (Status == EFI_UNSUPPORTED) {
+        AssertionType = EFI_TEST_ASSERTION_WARNING;
   } else {
     AssertionType = EFI_TEST_ASSERTION_FAILED;
   }
@@ -196,6 +202,8 @@ BBTestUpdateCapsuleConformanceTest (
   	             );
   if (Status == EFI_INVALID_PARAMETER) {
   	AssertionType = EFI_TEST_ASSERTION_PASSED;
+  } else if (Status == EFI_UNSUPPORTED) {
+        AssertionType = EFI_TEST_ASSERTION_WARNING;
   } else {
     AssertionType = EFI_TEST_ASSERTION_FAILED;
   }
